libstuff/libstuff.cpp

```diff
@@ -2563,11 +2563,7 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     string sqlToLog = sql;
     if ((int64_t)elapsed > warnThreshold) {
         // This code removing authTokens is a quick fix and should be removed once https://github.com/Expensify/Expensify/issues/144185 is done.
-        string match;
-        const bool hasAuthToken = SREMatch(".*(\"authToken\"\\:\"[0-9A-F]{400,1024}\").*", sql, match);
-        if (hasAuthToken) {
-            sqlToLog = SReplace(sql, match, "<REDACTED_AUTHTOKEN>");
-        }
+        pcrecpp::RE("\"authToken\":\"[0-9A-F]{400,1024}\"").GlobalReplace("\"authToken\":<REDACTED>", &sqlToLog);
```

Comment:
Wasn't there a story about pcrecpp being awfully slow and that's why we prefer `SREMatch` ? or am I misremembering this?

Reply:
SREMatch is pcre :)
https://github.com/Expensify/Bedrock/blob/master/libstuff/libstuff.cpp#L2679-L2685

std::regex is slow (or was hen we last tried it) which is why we prefer pcre.